alerts/openshift-container-storage-operator/StorageAutoScalingFailed.md

```diff
@@ -0,0 +1,76 @@
+# StorageAutoScalingFailed
+
+## Meaning
+
+During automatic storage scaling, failures may occur due to various factors. In
+such instances, the `StorageAutoScalingFailed` alert is fired.
+
+## Impact
+
+The storage cannot be scaled at the moment.
+
+## Diagnosis
+
+There are multiple stages where the storage configurations might not have matched
+the requested storage parameters.
+
+1. OSD pod is in `Pending` state:
+    If it is in `Pending` state with the following logs, there is a high chance
+    it is due to the exhaustion of the resources.
+    ```bash
+    Warning  FailedScheduling  3m53s  default-scheduler  0/6 nodes are available: 1 Insufficient cpu, 2 node(s) didn't match Pod's node affinity/selector, 3 node(s) had untolerated taint {node-role.kubernetes.io/master: }. preemption: 0/6 nodes are available: 1 Insufficient cpu, 5 Preemption is not helpful for scheduling.
```

Comment:
This warning is a result of describing the pod with the command below on line 26. Need to change the order so the output comes after the command?